### ISSUE TITLE
How do I set up aim assist?

### ISSUE THREAD

**Issue by kdias7** [Jire/Charlatano#718]
Recently I turned off the activate from fire key to look more legit, but i cant even tell if aim assist is there, do I have to hold the aim key for it to work? Also, if i put the aim assist strictness higher, does that mean that it looks more legit or less legit?

**Comment by Glloki** [comment]
If u put higher it will be more legit if u let it to 1.0 will be less legit

**Comment by kdias7** [comment]
Im talking about the aim assist strictness not just the aimbot strictness

**Comment by skorpek** [comment]
yeah I'm confused about this also, I would like aim assist to help a little bit with my aim but I can't find the right settings for this

**Comment by skorpek** [comment]
if you change the aim speed min and max of the normal aimbot, does the aim speed change? in my case it does.

**Comment by LonelyLinear** [comment]
> it's 26-32 but aim-assist strictness basically changes the aim assist speed. and so does aim-speed-min/max. what is the difference between those 2

good question. I'll try to find out, and if I'm lucky enough to get the results, I'll post it here

**Comment by LonelyLinear** [comment]
> it's 26-32 but aim-assist strictness basically changes the aim assist speed. and so does aim-speed-min/max. what is the difference between those 2

OK, so after I did series tests, I found out that speed is speed, the higher the value, the slower the speed(but if the speed too fast, even if the aim-assist strictness is 100, you can't aim the head before the next shot). You can think of the head as a magnetic target, so the aim-assist strictness is used to control the strength of its magnetism(btw, I think it works by distance, and the maximum value should only be set below 100...)so in other words, the aim-assist strictness could influence the speed of aiming, but it is much more important than aim-speed. I hope you understand what I'm saying...sorry for my bad English.
